packages/services/policy/src/api.ts

```diff
@@ -64,10 +64,10 @@ export const schemaPolicyApiRouter = t.router({
   availableRules: procedure.query(() => {
     return RELEVANT_RULES.map(([name, rule]) => ({
       name,
-      description: rule.meta.docs?.description || '',
-      recommended: rule.meta.docs?.recommended ?? false,
-      url: rule.meta.docs?.url,
-      schema: normalizeAjvSchema(rule.meta.schema) as object | null,
+      description: rule.meta!.docs!.description || '',
+      recommended: rule.meta!.docs!.recommended ?? false,
+      url: rule.meta!.docs!.url,
+      schema: normalizeAjvSchema(rule.meta!.schema),
```

Comment:
Please avoid using `!`. If you need to use it add a comment on why it is necessary or safe to use it here.

Reply:
fixed